setup.py

```diff
@@ -4,9 +4,6 @@
 
 setup(
     ext_modules = cythonize([
-    Extension("pyfsuipc", ["pyfsuipc.pyx"],
-              include_dirs=['./include'],
-              library_dirs=['./lib'],
-              libraries=["FSUIPC_User"])
+    Extension("pyfsuipc", sources=["pyfsuipc.pyx", "IPCuser.c"], libraries=["advapi32", "user32", "kernel32", "ole32", "oleaut32", "gdi32", "gdiplus", "imm32"])
```

Comment:
In the spirit of [PEP-8](https://www.python.org/dev/peps/pep-0008/#maximum-line-length), line length should not exceed 99 chars.